mix.lock

```diff
@@ -2,20 +2,20 @@
   "castore": {:hex, :castore, "1.0.6", "ffc42f110ebfdafab0ea159cd43d31365fa0af0ce4a02ecebf1707ae619ee727", [:mix], [], "hexpm", "374c6e7ca752296be3d6780a6d5b922854ffcc74123da90f2f328996b962d33a"},
```

Comment:
Btw, if you run `mix deps.unlock --unused`, that should remove :castore dependency. Since OTP 25 we have access to OS trust store so this is no longer necessary! (If you use Docker, make sure your images have ca-certificates or similar packages.) On CI you can check if you don't have unused deps with `mix deps.unlock --check-unused`.